action.py

```diff
@@ -32,11 +40,19 @@ class ContainerConfig:
     ports: tuple[PortMapping, ...]
     memory: int = 1024
     links: tuple[str, ...] = ()
+    mount_points: list[ContainerMountPoint] = field(default_factory=list)
```

Comment:
```suggestion
    mount_points: tuple[ContainerMountPoint, ...] = ()
```
Für die interne Verarbeitung kann das ein Tuple sein, oder? Nur beim Rausschreiben muss es eine Liste sein.